mscore/scoretab.h

```diff
@@ -52,6 +52,8 @@ class ScoreTab : public QWidget {
       QStackedLayout* stack;
       MuseScore* mainWindow;
       void clearTab2();
+      TabScoreView* tabScoreView(int idx) { return static_cast<TabScoreView*>(tab->tabData(idx).value<void*>()); }
```

Comment:
Can we avoid `static_cast` conversions here? What could be done to avoid this?

Reply:
Since an object returned stored in `QTabBar` and returned by `tabData()` is `QVariant` I believe the only way to avoid `static_cast` here is to register `TabScoreView` structure in Qt's meta-object system. Should we do it here?